apps/arweave/src/ar_http_iface_client.erl

```diff
@@ -123,9 +124,11 @@ add_peer(Peer) ->
 
 %% @doc Get a peers current, top block.
 get_current_block(Peer) ->
-	get_current_block(Peer, get_block_index(Peer)).
-get_current_block(Peer, BI) ->
-	case get_full_block([Peer], hd(BI), BI) of
+	get_current_block(Peer, ?WITH_TX_DATA).
```

Comment:
Could we get rid of this method along with the part of `ar_node:get_current_block/1` that calls it? As far as I can see, `ar_http_iface_client:get_current_block/..` is only used by `ar_node:get_current_block`, which in turn is only used once in `ar_http_iface_tests` (when an IP is passed, not a PID).